### ISSUE TITLE
Zoom G5 findings

### ISSUE THREAD

**Issue by therootlord** [mungewell/zoom-zt2#4]
Hey bro, mine is a G5, i have inspected its firmware, and i found some common patterns probably regarding to the effects data, i have also compared those with some zdl patches and the "core code" seems very similar, they stripped hardly the symbols, debug info etc.. from the firmwares of the old models, so its difficult to associate things). 
Do you have interest on trying something for the old models? Thanks!

**Comment by clint-toy** [comment]
Have you tried Barsik-Barbosik/Zoom-Firmware-Editor?

**Comment by therootlord** [comment]
Yep, it didn't load the firmware, since it doesn't have the name conventions as the new models

**Comment by shooking** [comment]
hi @therootlord 
I got me a Zoom G5 (as well as a G5n).
I am making quite a bit of process remote controlling it - need to because the buttons are broke.
You can find info on my YouTube and my GitHub - https://github.com/shooking/ZoomPedalFun

I havent found how to inject new FX yet - for sure they are neither ZDLs nor ZD2s.
But it has enough character that I dont care - the Z plane pedal is sweet and it's built like a tank.
I am trying to get my head around the pedal SW params - on my site I created a wiki and added anything I found so far.
If you are still interested please hit me up with any ideas of what you want to explore with them,
